Remotion/Web/Core/UI/Controls/WebTab.cs

```diff
@@ -33,13 +33,13 @@ namespace Remotion.Web.UI.Controls
 public class WebTab: IWebTab, IControlStateManager
 {
   /// <summary> The control to which this object belongs. </summary>
-  private IControl _ownerControl;
+  private IControl? _ownerControl;
   private string _itemID = "";
   private string _text = "";
   private IconInfo _icon;
   private bool _isVisible = true;
   private bool _isDisabled;
-  private WebTabStrip _tabStrip;
+  private WebTabStrip? _tabStrip;
   private bool _isSelected;
   private int _selectDesired;
   private bool _isControlStateRestored;
```

Comment:
We should make IconInfo nullable in the ctor on line 48 and then default-initialize in the body with new IconInfo(string.Empty). When doing this change, we need to verify that the behavior (i.e. rendering of the tabs) remains unchanged.